packages/babel-plugin-proposal-explicit-resource-management/test/fixtures/transform-sync/named-evaluation/output.js

```diff
@@ -0,0 +1,16 @@
+let log = [];
+let Symbol$dispose = Symbol.dispose || Symbol.for("Symbol.dispose");
+try {
+  var _m;
+  var _usingCtx = babelHelpers.usingCtx();
+  const foo = _usingCtx.u((_m = class {
+    static [Symbol$dispose]() {
+      log.push(`${foo.name} is disposed`);
+    }
+  }, babelHelpers.setFunctionName(_m, "foo")));
```

Comment:
`_usingCtx.u(babelHelpers.setFunctionName(class {}, "foo")))`
Is there any difference with this?